helm/minio-operator/values.yaml

```diff
@@ -23,3 +23,52 @@ console:
     tag: v0.6.3
     pullPolicy: IfNotPresent
   replicaCount: 1
+  secrets:
```

Comment:
This console section is for operator console. These secrets have to be added to a new `console` section under tenants